engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -23,7 +23,9 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
-    private static SetWireframe defaultInstance = new SetWireframe(false);
+    private static final boolean DEFAULT_VALUE = false;
```

Comment:
Please consider #2421 on this file. Changes in this PR will be overridden by that.